tests/devapps/WAM/NetCoreWinFormsWam/Form1.cs

```diff
@@ -320,11 +322,12 @@ private async Task<AuthenticationResult> RunAtiAsync(IPublicClientApplication pc
 
             AuthenticationResult result = null;
             var scopes = GetScopes();
-
+            var guid = Guid.NewGuid();
             var builder = pca.AcquireTokenInteractive(scopes)
                 .WithUseEmbeddedWebView(true)
                 //.WithExtraQueryParameters("domain_hint=live.com") -- will force AAD login with browser
                 //.WithExtraQueryParameters("msafed=0")             -- will force MSA login with browser
+                .WithProofOfPossession(System.Net.Http.HttpMethod.Get, new Uri( pca.Authority), "eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsImtpZCI6bnVsbH0.eyJ0cyI6MTY1MjI4NTAzNH0.Nh-mAJwRphv57IdpdIrYzmYp6vP_BmmYy4UrNKj5A2x4XKLbp_H3aH4J5_s9hP5MzoiHE2SgVaDG8YUbP4xOjFYmpNG884pWqI-z9RjFNKJgBTXUhwv8HsUnxUHq1KTvpLmd1K1gJZORdeUI2LDr07EEH3-aT0PkRt-wT1YNNh5gU_RHV5KvlsyDWCvCJpEbZmGUf8JX9tHO2ux7XAKD77lVb5m6lFq_8Wr5nhJDyREHrXKWQq-X4rTxnBCZ4KBAufImSVHAeVi7ihlGbcobU2CuyJscTZkyELWMG8rBD6QK57AzrM77mua9-QClKIHArL8_d2fgyksLLS89wxy25A")
```

Comment:
We need fancy UI this :). I'm joking, but we do need at least a checkbox or dropdown to select between Bearer and POP.

Reply:
Yes, next update will fix